src/Classes/ListingHydrator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace ListingsApp\Classes;
+require_once '../../vendor/autoload.php';
+
+use PDO;
+
+/** connects to database
+ * @return PDO
+ */
+
+function getDB(): PDO
+{
+    return new PDO('mysql:host=db;dbname=listings-feed', 'root', 'password');
+
+}
+/**
+ * Class ListingHydrator
+ * @package ListingsApp\Classes
+ */
+
+class ListingHydrator
+{
+    public static function getForSaleListings(PDO $db): array
+    {
+        $query = $db->prepare('SELECT * FROM `listings` WHERE `type` = 1;');
+        $query->execute();
+        $query->setFetchMode(PDO::FETCH_CLASS, Listing::class );
+        return $query->fetchAll();
+    }
+
+    public static function getForLetListings(PDO $db): array
```

Comment:
Missing docblocks

Reply:
Also any thoughts on combining these two methods into one? Not something you have to do but just a suggestion/thought!

Reply:
yep makes sense!